oemer/build_system.py

```diff
@@ -114,27 +117,27 @@ def __repr__(self):
 
 
 class Measure:
-    def __init__(self):
-        self.symbols = []  # List of symbols
-        self.double_barline: bool = None
+    def __init__(self) -> None:
+        self.symbols: List[Any] = []  # List of symbols
+        self.double_barline: Union[bool, None] = None
         self.has_clef: bool = False
         self.clefs: list[Clef] = []
         self.voices: list[NoteGroup] = []
         self.sfns: list[Sfn] = []
         self.rests: list[Rest] = []
-        self.number: int = None
-        self.at_beginning: bool = None
-        self.group: int = None
+        self.number: Union[int, None] = None
+        self.at_beginning: bool = None  # type: ignore
+        self.group: Union[int, None] = None
 
-        self.time_slots: list[object] = []
-        self.slot_duras: np.ndarray = None
+        self.time_slots: list[List[Any]] = []
+        self.slot_duras: np.ndarray = None  # type: ignore
```

Comment:
Better to annotate as `Union[np.ndarray, None]` and remove the type ignore?

Reply:
I investigated some cases, but honestly I don't think I can spend the time on this PR which would be needed to improve the None handling. I could help a bit on that topic, but it looks like a lot of work.